advanced-usage/index.md

```diff
@@ -44,7 +44,7 @@ pane, which can be hovered over to access a scrollable history of alerts:
 
 Some alerts can be read aloud on arrival using text to speech technology, which [can be configured](#alerts).
 
-#### Mini Widget Containers
+#### Mini Widget Bar
```

Comment:
Those are indeed containers.
The bar is just the floating widget one (which is badly named, as it's not a bar haha).